src/Native/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+project(System.IO.Compression.Native)
```

Comment:
Currently all of the source under src/Native is Unix-specific.  This PR is adding Windows code as well, but there's nothing to differentiate it.  I suggest before we do this PR that we move the code around to ensure a proper separation, e.g. src/Native/Windows/LibraryName,src/Native/Unix/LibraryName, etc.  Without that, for example, it looks like this PR is going to conflict with @eerhardt's PR that's also adding a System.IO.Compression.Native, except his is for Unix.